xla/service/gpu/model/gpu_performance_model_base.cc

```diff
@@ -109,6 +109,17 @@ std::optional<absl::Duration> GpuPerformanceModelCache::Get(
   return std::nullopt;
 }
 
+const absl::flat_hash_map<const HloInstruction*, absl::Duration>&
+GpuPerformanceModelCache::GetAllConsumers(const HloInstruction& producer) {
+  auto it = fusion_runtime_data_.find(&producer);
+  return it->second;
+}
+
+bool GpuPerformanceModelCache::Contains(const HloInstruction& producer) {
```

Comment:
There are also cache entries in `instruction_runtime_data_`, but this function only checks `fusion_runtime_data_`. We should probably make the function name more specific which cache entries we check.

Reply:
I used `ContainsConsumers`.